dataclass_csv/dataclass_reader.py

```diff
@@ -13,13 +13,13 @@ class DataclassReader:
     def __init__(
         self,
         f,
-        cls,
-        fieldnames=None,
-        restkey=None,
-        restval=None,
-        dialect='excel',
-        *args,
-        **kwds,
+        cls: Type[object],
```

Comment:
I think it needs to run black: `black -l79 -S ./dataclass_csv`. It will format other parts of the code but it is good to do it.

Reply:
Yeah I ran black per contributing and it hit so much of the code I wasn't willing to submit a bunch of unrelated changes. Happy to do so with your blessing though. :)